src/FormatUtils/userData.ts

```diff
@@ -26,41 +26,35 @@ export const formatBadgesData = (data: UserData) => ({
 });
 
 export const formatContestData = (data: UserData) => ({
-  contestAttend: data.userContestRanking.attendedContestsCount,
-  contestRating: data.userContestRanking.rating,
-  contestGlobalRanking: data.userContestRanking.globalRanking,
-  totalParticipants: data.userContestRanking.totalParticipants,
-  contestTopPercentage: data.userContestRanking.topPercentage,
-  contestBadges: data.userContestRanking.badge,
+  contestAttend: data.userContestRanking?.attendedContestsCount,
+  contestRating: data.userContestRanking?.rating,
+  contestGlobalRanking: data.userContestRanking?.globalRanking,
+  totalParticipants: data.userContestRanking?.totalParticipants,
+  contestTopPercentage: data.userContestRanking?.topPercentage,
+  contestBadges: data.userContestRanking?.badge,
```

Comment:
Here is the change I made to the code.

For example in my user profile data.userContestRanking is `null` so the program was erroring out here and returning an empty object because data.userContestRanking was inexistent. Now with the logical operator `?` it instead skips those attribute assignments and just returns `{ contestParticipation: [] }`

Reply:
Ok understand